### PR TITLE
Dashboard fix enhancement

### PR DESCRIPTION
**Pull request by mesrivatsav**
Check the changes and push the commits. I'm attaching a screenshot for reference
Thank you
screencapture-localhost-3001-2020-04-22-02_33_31

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { socketUrl } from "./configs/env";
 import SocketContext from "./context/socket";
 import Dashboard from "./containers/dashboard";
 import io from "socket.io-client";
+import 'bootstrap/dist/css/bootstrap.min.css';
```
Comment (Impeekay) [inline src/App.js:7]: are we using bootstrap?

File: src/components/dashboard.js [hunk 2]
```diff
@@ -1,155 +1,206 @@
...
   Segment,
   SegmentGroup,
   Header,
-  Checkbox,
-  Label,
-  Table,
   Dimmer,
   Loader,
   Message
 } from "semantic-ui-react";
```
Comment (Impeekay) [inline src/components/dashboard.js:9]: if we're using bootstrap, no need of semantic UI, use either of the two. Not both

**Comment by mesrivatsav** [comment]
Okay i will change that

File: src/components/dashboard.js [hunk 3]
```diff
@@ -1,155 +1,206 @@
...
 import DatePicker from "react-datepicker";
-// import JSONPretty from "react-json-pretty";
+import moment from 'moment'
+// import { render } from 'react-dom';
+import 'bootstrap/dist/css/bootstrap.min.css';
```
Comment (Impeekay) [inline src/components/dashboard.js:13]: Not necessary since you've imported this in `App.js`

File: src/components/dashboard.js [hunk 4]
```diff
@@ -1,155 +1,206 @@
...
+//import $ from 'jquery';
+// import Popper from 'popper.js';
+import 'bootstrap/dist/js/bootstrap.bundle.min';
```
Comment (Impeekay) [inline src/components/dashboard.js:16]: Move this import also to `App.js` like that you'll have access to it everywhere in the child components.

File: src/containers/dashboard.js [hunk 5]
```diff
@@ -2,6 +2,15 @@ import React from "react";
 import { fetchPiData } from "../api/piData";
 import DashboardComponent from "../components/dashboard";
 import moment from "moment";
+import 'bootstrap/dist/css/bootstrap.min.css';
```
Comment (Impeekay) [inline src/containers/dashboard.js:5]: same remove this import

**Comment by mesrivatsav** [comment]
Yeah fine i will do the necessary changes and push the commits

**Comment by mesrivatsav** [comment]
Made the necessary changes check once

**Pull request merged**